config/postcss-add-fallback.js

```diff
@@ -1,9 +1,16 @@
 
-//replace with version from ui5-web-components when we take dependency on them
 const postcss = require('postcss');
 const fs = require('fs');
 const arrayUniq = require('array-uniq');
 
+const supportedThemes = [
+    'sap_fiori_3',
+    'sap_fiori_3_dark',
+    'sap_fiori_3_hcb',
+    'sap_fiori_3_hcw',
+    'sap_fiori_3_light_dark'
```

Comment:
Are we doing "Light Dark"?  Is this a real thing? 😆

Reply:
I know 😆  but yes

Reply:
taking over under  💵 on when `dark_light` appears...